app/index.js

```diff
@@ -7,11 +6,11 @@ const TwoByTwoCards = React.createClass({
   render() {
     return (
       <View style={[styles.container, {flexDirection: 'row' }]}>
```

Comment:
**<a href='http://eslint.org/docs/rules/object-curly-spacing'>object-curly-spacing</a>**: A space is required before '}'